src/components/canvas/connections/BatchPath2D/index.tsx

```diff
@@ -9,40 +11,56 @@ export interface Path2DRenderInstance {
   getPath(): Path2D | undefined | null;
   style(ctx: CanvasRenderingContext2D): Path2DRenderStyleResult | undefined;
   afterRender?(ctx: CanvasRenderingContext2D): void;
+  isPathVisible?(): boolean;
 }
 
-class Path2DGroup {
+const CHUNK_SIZE = 100;
```

Comment:
Just an idea: maybe we should put this as a setting for the [graph](https://github.com/gravity-ui/graph/blob/main/src/graphConfig.ts#L90)? Then the user will be able to adjust this value based on performance requirements.